lte/gateway/c/oai/tasks/amf/deregistration_request.cpp

```diff
@@ -263,9 +246,9 @@ void amf_remove_ue_context(
           LOG_AMF_APP, ue_context_p->amf_context._imsi64,
           "UE context not found for n11_ue_context\n"
           " gnb_ue_ngap_id "
-          "0x%08" PRIX64
+          "0x%08" PRIX32
```

Comment:
Here and other places in this file, please use GNB_UE_NGAP_ID_FMT and AMF_UE_NGAP_ID_FMT

Reply:
fixed